data/pipeline/scripts/locations/parse_congress_districts.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# takes congress_members.html from stdin, and writes to stdout a CSV of Colorado
+# congress members.
+# each row includes the district number, member name, affiliation, and website
+# (website is derived from the member's name)
+
+# use hq to extract table with CSS class name "library-table"
+# then use jq (in several steps; optimize later?) to extract the data, annotate
+# with some computed fields, and format it as CSV
+hq 'array {
+    //table//tr ->
+    hash {
+        member: $_/td[1]//span[1] | text,
+        affiliation: $_/td[2] | text,
+        state: $_/td[3] | text,
+        district: $_/td[4] | text
+    }
+}' \
+    | jq '[ .[] | select(.state == "Colorado (CO)") | . + {district: .district | scan("[0-9]+") | tonumber} ] | sort_by(.district)' \
+    | jq -r '(
+        ["DistrictNum", "Member", "Affiliation", "Website"],
+        (
+            .[] | [
+                .district,
+                .member,
+                .affiliation,
+                "https://\(.member | split(", ") | .[0] | ascii_downcase ).house.gov/"
```

Comment:
It's cool they have dedicated subdomain websites for each of them, though very minor comment, idk how they handle duplicate last names, of which there are several. Maybe we should just use the links in the table instead? Not a big deal at all; I can't remember if I'm even showing this URL in the frontend.

Reply:
Good point; I at first was building those URLs directly to their personal websites because that's what Cydney had included in her original spreadsheet. The pages linked in the table are IMHO more actionable anyway, and there's a link to the `<member>.house.gov` site on each one anyway. I'll go ahead and just include the links from the table, then.